src/tests/ftest/container/open.py

```diff
@@ -27,106 +27,141 @@
 import traceback
 import uuid
 
-from avocado import main
 from apricot import TestWithServers
+from test_utils import TestPool, TestContainer
+from avocado.core.exceptions import TestFail
 
-from daos_api import DaosPool, DaosContainer, DaosApiError
-
+RESULT_PASS = "PASS"
+RESULT_FAIL = "FAIL"
+RESULT_TO_NUM = {RESULT_PASS: 0, RESULT_FAIL: 1}
 
 class OpenContainerTest(TestWithServers):
     """
-    Tests DAOS container bad create (non existing pool handle, bad uuid)
-    and close.
+    Tests container's open function. Create 2 pool-container pairs and test
+    the following.
+    1. Use valid pool handle and container UUID. Expected to pass
+    container1.open(pool1.handle, container1_uuid)
+    container2.open(pool2.handle, container2_uuid)
+    2. Use the other container's UUID. Expected to fail
+    container1.open(pool1.handle, container2_uuid)
+    container2.open(pool2.handle, container1_uuid)
+    3. Use the other pool's handle. Expected to fail
+    container1.open(pool2.handle, container1_uuid)
+    container2.open(pool1.handle, container2_uuid)
```

Comment:
We should implement the documentation format defined in the global validation test plan:

```
class <TestClass>(TestWithServers):
    """
    Test Class Description:
        <An overview of all the tests in this python class.>

    :avocado: recursive
    """

    def test_<test_name>(self):
        """Jira ID: <DAOS-xxxx>.

        Test Description:
             <description specific to this test case>

        Use Cases:
             <list of use cases that are to at least some degree covered by this test case>

        :avocado: tags=<tags>
        """
```

Reply:
I see. Thanks for the template.